source/components/AvatarStack/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Avatar from '../Avatar';
+
+import './styles.scss';
+
+/* eslint-disable react/no-array-index-key */
+const AvatarStack = ({ avatars, overrideCount, maxStackSize, hideOverflow, className }) => {
```

Comment:
A little late in the game now but it would be great if this were a separate PR. I would rather review three smaller contained PRs than one big one

Reply:
It's already on master, I've just forget to update it here.